lib/Fhp/Model/StatementOfAccount/Transaction.php

```diff
@@ -54,25 +59,49 @@ class Transaction
     protected $name;
 
     /**
+     * Get booking date
+     *
+     * @return \DateTime|null
+     */
+    public function getBookingDate()
+    {
+        return $this->bookingDate;
+    }
+
+    /**
      * Get date
      *
      * @return \DateTime|null
      */
-    public function getDate()
```

Comment:
Please do not delete the method "getDate".
As it is public, someone else could use it already in his code.
Better would be to return the booking date on "getDate" and just mark the method as deprecated.
So people have time to change their code to use one of the new methods.
Thanks :)